sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/mathExpressions.scala

```diff
@@ -291,10 +291,6 @@ case class Cosh(child: Expression) extends UnaryMathExpression(math.cosh, "COSH"
   usage = """
     _FUNC_(expr) - Returns inverse hyperbolic cosine of `expr`.
   """,
-  arguments = """
-    Arguments:
-      * expr - hyperbolic angle
```

Comment:
This description is wrong; the argument is not an angle. For consistency with other inverse functions, I just removed the description.

Reply:
Oops. Thanks for the fix.